### PR TITLE
kraken: tests: upgrade/hammer-x failing with OSD has the store locked when Thrasher runs ceph-objectstore-tool on down PG

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/19558

### CODE REVIEW

**Comment by smithfarm** [comment]
@tchaikov This passed a rados suite at http://tracker.ceph.com/issues/19009#note-75

Please review.

**Comment by smithfarm** [comment]
@tchaikov @jdurgin @gregsfortytwo This passed an `upgrade/jewel-x` suite at http://tracker.ceph.com/issues/19009#note-76 and a rados suite at http://tracker.ceph.com/issues/19009#note-75

Please review.

**Comment by smithfarm** [comment]
@yuriw This makes the upgrade suites behave better - please review/merge before starting QE.

File: qa/tasks/ceph_manager.py [hunk 1]
```diff
@@ -241,12 +241,22 @@ def kill_osd(self, osd=None, mark_down=False, mark_out=False):
                         break
                     log.debug("ceph-objectstore-tool binary not present, trying again")
 
-            proc = exp_remote.run(args=cmd, wait=True,
-                                  check_status=False, stdout=StringIO())
-            if proc.exitstatus:
-                raise Exception("ceph-objectstore-tool: "
-                                "exp list-pgs failure with status {ret}".
-                                format(ret=proc.exitstatus))
+            # ceph-objectstore-tool might bogusly fail with "OSD has the store locked"
```
Comment (amitkumar50) [inline qa/tasks/ceph_manager.py]: @smithfarm can you please keep title of Future PRs within 50 characters..
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits

**Comment by smithfarm** [comment]
Kraken is EOL.

**Pull request closed without merging**